src/app/components/Billboard/index.styles.tsx

```diff
@@ -0,0 +1,88 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../../../src/app/utilities/pixelsToRem';
+
+export default {
+  clickAreaContainer: () =>
```

Comment:
As mentioned in https://github.com/bbc/simorgh/pull/11489/files#r1562151617

Let's rename this, and add link specific styling